src/dg-testVault/.obsidian/workspace.json

```diff
@@ -28,63 +28,62 @@
             }
           },
           {
-            "id": "4384912f0352e092",
+            "id": "15581449e40a9e45",
             "type": "leaf",
             "state": {
               "type": "markdown",
               "state": {
-                "file": "001 Links.md",
+                "file": "000 Home.md",
+<<<<<<< Updated upstream
```

Comment:
I see there are som remains from a stash merge. Could you fix those?

Reply:
sure! I do everything with rebase, I like how it leaves a clean history and rarely get rebase issues. (but it changes history more which can be weird and requires branch protection to make sure you can't destroy the entire repo if you have write access :D) 

ps do you rebase/merge in vscode? vscode yells at me if i try to add with merge conflicts

Reply:
I'll see how this happened :D, maybe let's gitignore the file, or is it useful?

Reply:
I gitignored it for now, let's see if we need it? It seems like it could be useful, but it's also an absolute pain.

Reply:
Yes, I've also had issues with it when merging stuff the last couple of days. Lets try ignoring. We can always bring it back if it turn out we need it